journal/modernized/08-chapter-7.adoc

```diff
@@ -2,6 +2,7 @@
 
 Testimony Of the Monthly Meeting of Friends of Philadelphia for the Northern District,
 concerning William Savery.
+//Does this need a sub-heading or anything?
```

Comment:
yeah, lets use a [.blurb] for this. LIke this:

```ASCIIDOC
[.blurb]
=== Testimony of the Monthly blah blah
```

both here and in the original.